src/main/java/com/tickets/controller/CinemaHallController.java

```diff
@@ -27,15 +27,15 @@ public CinemaHallController(CinemaHallService cinemaHallService,
 
     @PostMapping
     public void addCinemaHall(@RequestBody CinemaHallRequestDto cinemaHallRequestDto) {
-        CinemaHall cinemaHall = cinemaHallMapper.dtoToMap(cinemaHallRequestDto);
+        CinemaHall cinemaHall = cinemaHallMapper.mapToEntity(cinemaHallRequestDto);
         cinemaHallService.add(cinemaHall);
     }
 
     @GetMapping
     public List<CinemaHallResponseDto> getAll() {
         return cinemaHallService.getAll()
                 .stream()
-                .map(cinemaHallMapper::mapToDto)
+                .map(cinemaHallMapper::entityToMap)
```

Comment:
what does `entityToMap` mean? are we creating a Map from Entity?
```suggestion
                .map(cinemaHallMapper::mapToDto)
```